newIDE/app/src/GameDashboard/LeaderboardAdmin/MaxLeaderboardCountAlertMessage.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import * as React from 'react';
+import { Trans } from '@lingui/macro';
+
+import Text from '../../UI/Text';
+import RaisedButton from '../../UI/RaisedButton';
+import AlertMessage from '../../UI/AlertMessage';
+import { Line, Column } from '../../UI/Grid';
+import { type Limits } from '../../Utils/GDevelopServices/Usage';
+
+type Props = {|
+  onUpgrade: () => void,
+  onClose: () => void,
+  limits: Limits,
+|};
+
+const MaxLeaderboardCountAlertMessage = ({
+  onUpgrade,
+  onClose,
+  limits,
+}: Props) => {
+  const {
+    maximumCountPerGame,
+    canMaximumCountPerGameBeIncreased,
+  } = limits.capabilities.leaderboards;
+
+  return (
+    <Line>
+      <Column expand>
+        <AlertMessage
+          kind="warning"
+          onHide={onClose}
+          renderRightButton={
+            canMaximumCountPerGameBeIncreased
+              ? () => (
+                  <RaisedButton
+                    primary
+                    label={<Trans>Check our premiums plans</Trans>}
+                    onClick={onUpgrade}
+                  />
+                )
+              : undefined
+          }
+        >
+          <Text size="block-title">
+            <Trans>
+              You've reached your maximum of {maximumCountPerGame} leaderboards
+              for your game
```

Comment:
nit: `.` at the end

Reply:
It's a title so I'm not sure it's necessary. Wdyt?

Reply:
oh right, didn't see that, all good